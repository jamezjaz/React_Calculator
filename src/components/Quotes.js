import React from 'react';

const Quotes = () => (
  <div>
    <p>Pure mathematics is, in its way, the poetry of logical ideas. — Albert Einstein</p>
    <p>
      Without mathematics, there’s nothing you can do. Everything around you is mathematics.
      Everything around you is numbers. — Shakuntala Devi
    </p>
    <p>
      What is mathematics? It is only a systematic effort of solving puzzles posed by nature.
      — Shakuntala Devi
    </p>
    <p>Mathematics is the music of reason. — James Joseph Sylvester</p>
  </div>
);

export default Quotes;
