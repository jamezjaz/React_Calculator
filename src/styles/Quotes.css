.quotes {
  font-size: 20px;
  padding: 20px;
  line-height: 1.5;
}
