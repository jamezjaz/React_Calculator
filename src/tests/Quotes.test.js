import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../components/Quotes';

it('renders correctly', () => {
  const tree = renderer
    .create(<Quotes />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});