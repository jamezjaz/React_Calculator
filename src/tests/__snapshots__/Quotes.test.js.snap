// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="quotes"
>
  <p>
    * Pure mathematics is, in its way, the poetry of logical ideas.
    <strong>
       — Albert Einstein
    </strong>
  </p>
  <p>
    * Without mathematics, there’s nothing you can do. Everything around you is mathematics. Everything around you is numbers.
    <strong>
       — Shakuntala Devi
    </strong>
  </p>
  <p>
    * What is mathematics? It is only a systematic effort of solving puzzles posed by nature.
    <strong>
       — Shakuntala Devi
    </strong>
  </p>
  <p>
    * Mathematics is the music of reason.
    <strong>
       — James Joseph Sylvester
    </strong>
  </p>
</div>
`;
